drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/domelements/ScenarioHeaderTextAreaDOMElement.java

```diff
@@ -44,6 +47,13 @@ public void setScenarioHeaderMetaData(ScenarioHeaderMetaData scenarioHeaderMetaD
 
     @Override
     public void flush(final String value) {
+        if (Objects.isNull(value) || value.isEmpty()) {
```

Comment:
Shouldn't we rather call `value.trim().isEmpty()` ?

Reply:
@jomarko You're right. Done.